KarhooSDKIntegrationTests/Service/Trip/BookTripMethodSpec.swift

```diff
@@ -35,7 +35,7 @@ final class BookTripMethodSpec: XCTestCase {
                                 locale: "")
 
         let passengers = Passengers(additionalPassengers: 0, passengerDetails: [PassengerDetails(user: userInfo)])
-        let tripBooking = TripBooking(quoteId: "123", passengers: passengers)
```

Comment:
Same with my above comment, this needs to be added back. This is still required.